### PR TITLE
Fixed UDP6 at deviceManagement.

### PR DESCRIPTION
**Pull request by ezequiel-umu**

### CODE REVIEW

**Comment by dmoranj** [comment]
Fixes #72

**Comment by dmoranj** [comment]
Fixes #71

File: config.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ config.server = {
     lifetimeCheckInterval: 1000,        // Minimum interval between lifetime checks in ms
     udpWindow: 100,
     defaultType: 'Device',
-    logLevel: 'FATAL',
+    logLevel: 'DEBUG',
```
Comment (dmoranj) [inline config.js]: Not really needed, let it as it was originally, please.

**Comment by dmoranj** [comment]
Add two lines to the CHANGES_NEXT_RELEASE file indicating issues #72 and #71 have been resolved, please.

**Comment by ezequiel-umu** [comment]
I think the conflicts come from CHANGES_NEXT_RELEASE and it should be trivial to merge.

**Comment by dmoranj** [comment]
Yes, it seems so. Just pull the changes from origin:develop into your local branch and merge them, commit and push. That should solve the conflicts.

For future contributions, it would be better if you also worked in your local `develop` branch, so we can easily merge from develop to develop.

**Comment by dmoranj** [comment]
LGTM. Sorry for the long time it took to merge.

**Pull request merged**